Validation/RecoTrack/python/TrajectorySeedValidation_cff.py

```diff
@@ -0,0 +1,40 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.RecoTrack.seedTracks_cfi import seedTracks as _seedTracks
+from SimTracker.TrackAssociation.trackingParticleRecoTrackAsssociation_cfi import trackingParticleRecoTrackAsssociation as _trackAssociation
+from Validation.RecoTrack.TrackValidation_cff import *
+
+_seedProducerLabels = ["initialStepSeeds",
+                       "detachedTripletStepSeeds",
+                       "lowPtTripletStepSeeds",
+                       "pixelPairStepSeeds",
+                       "mixedTripletStepSeedsA",
+                       "mixedTripletStepSeedsB",
+                       "pixelLessStepSeeds",
+                       "tobTecStepSeedsPair",
+                       "tobTecStepSeedsTripl",
+                       "jetCoreRegionalStepSeeds",
+                       ]
+
+_moduleNames = []
+for _label in _seedProducerLabels:
+    _lines = """
+{0}Tracks = _seedTracks.clone(src = cms.InputTag(\"{0}\"))
+{0}Association = _trackAssociation.clone(label_tr = cms.InputTag(\"{0}Tracks\"))
+{0}Validator = trackValidator.clone(
+   trackCollectionForDrCalculation = cms.InputTag("{0}Tracks"),
+   label = [cms.InputTag("{0}Tracks")],
+   associators = [cms.InputTag(\"{0}Association\")])
```

Comment:
Adding here `dodEdxPlots = False` will probably fix the ValueMap exception for FullSim (it is `False` by default, set to `True` for FullSim validation here
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/Validation/RecoTrack/python/TrackValidation_cff.py#L135
and back to `False` for FastSim validation here
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/Validation/RecoTrack/python/TrackValidation_fastsim_cff.py#L10
)